test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -182,8 +182,8 @@ func TestWorkspacesVirtualWorkspaces(t *testing.T) {
 							return false, nil
 						}
 						return false, err
-					} else if !conditions.IsTrue(cw, tenancyv1alpha1.WorkspaceURLValid) {
-						return false, fmt.Errorf("ClusterWorkspace %s is not valid: %s", cw.Name, conditions.GetMessage(cw, tenancyv1alpha1.WorkspaceURLValid))
+					} else if !conditions.IsTrue(cw, tenancyv1alpha1.WorkspaceShardValid) {
+						return false, fmt.Errorf("ClusterWorkspace %s is not valid: %s", cw.Name, conditions.GetMessage(cw, tenancyv1alpha1.WorkspaceShardValid))
```

Comment:
@sttts why is this a hard error, should not retry?
seens a failure here because of this https://github.com/kcp-dev/kcp/runs/5467154069?check_suite_focus=true